msteams-platform/whats-new.md

```diff
@@ -26,6 +26,7 @@ Teams platform features that are available to all app developers.
 
 **2024 January**
 
+* ***January 25, 2024***: [Teams domain migration to the new *.cloud.microsoft* domain.](m365-apps/domain-migration.md)
```

Comment:
I think this (future) breaking change announcement would fit best under Deprecated section (TeamsJS <12.19 versions will be in a "deprecated" state as of 6/1/24) of what's new

Reply:
How about:

***January 25, 2024***:  Teams, Outlook, and Microsoft 365 web domains are migrating to the consolidated *cloud.microsoft* domain. [Update your app](m365-apps/extend-m365-teams-personal-tab.md#configure-content-security-policy-headers) by June 1, 2024 to ensure it continues to run seamlessly in Teams, Outlook and Microsoft 365 web experiences.

Reply:
Where the link is:

 `[Update your app](m365-apps/extend-m365-teams-personal-tab.md#configure-content-security-policy-headers)`

Reply:
Done